lumigator/python/mzai/sdk/pyproject.toml

```diff
@@ -1,13 +1,13 @@
 [project]
-name = "sdk"
-version = "0.1.0"
+name = "lumigator-sdk"
+version = "0.1.0-rc.1"
 description = "Add your description here"
```

Comment:
Let's add a description for documentation completeness as long as we're working on this file and sending it to PyPI -it will show up here https://pypi.org/project/lm-buddy/#description

Reply:
Ok